### PR TITLE
ServiceTarget.FromEntities support

### PR DESCRIPTION
**Pull request by helto4real**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (which adds functionality to an existing integration)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [x] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] Local tests pass. **Your PR cannot be merged unless tests pass**
- [x] There is no commented out code in this PR.
- [x] I have followed the [development checklist][dev-checklist]
- [x] The code compiles without warnings (code quality chek)
- [x] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/net-daemon/docs

### CODE REVIEW

File: src/HassModel/NetDeamon.HassModel/Common/ServiceTarget.cs [hunk 1]
```diff
@@ -12,8 +13,16 @@ public class ServiceTarget
...
-            new() { EntityIds = new[]{ entityId } };
-        
+            new() { EntityIds = new[] { entityId } };
+
+        /// <summary>
+        /// Creates a new ServiceTarget from a list of EntityIds
+        /// </summary>
+        /// <param name="entityIds">The Ids of entities</param>
+        /// <returns>A new ServiceTarget</returns>
+        public static ServiceTarget FromEntities(IEnumerable<string> entityIds) =>
```
Comment (BeeHiveJava) [inline src/HassModel/NetDeamon.HassModel/Common/ServiceTarget.cs:23]: I suddenly got the idea to make another extension that takes a `params string[] entityIds` so you can easilly do something like: `FromEntities("entity1", "entity2", "entity3")`, should you want to.

Comment (BeeHiveJava) [reply src/HassModel/NetDeamon.HassModel/Common/ServiceTarget.cs:23]: But also how much effort is it to do `FromEntities(new [] { "entity1", "entity2" }` :smile:

**Review by BeeHiveJava — APPROVED** [review]

File: src/HassModel/NetDeamon.HassModel/Common/ServiceTarget.cs (hunk 1 above)
Comment (helto4real) [reply src/HassModel/NetDeamon.HassModel/Common/ServiceTarget.cs:23]: I can add that while I am at it

Comment (helto4real) [reply src/HassModel/NetDeamon.HassModel/Common/ServiceTarget.cs:23]: Yea but looks a bit cleaner though

**Review by BeeHiveJava — APPROVED** [review]

**Pull request merged**